micromasters/settings.py

```diff
@@ -557,6 +559,8 @@ def get_all_config_keys():
     in get_all_config_keys() if key.startswith(MM_FEATURES_PREFIX)
 }
 
+FEATURE_FLAG_QS_PREFIX = get_var("FEATURE_FLAG_QS_PREFIX", None)
```

Comment:
I'll rename this with a different prefix, it was unintentional that this pollute that namespace.